File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.js
const input = require('fs').readFileSync(0, 'utf8').toString().trim().split('\n');

const n = Number(input[0]);
const size_num = input[1].split(' ');
const [cloth, pen] = input[2].split(' ');

Number(size_num, cloth, pen);
let bundle = 0;

for(const size of size_num){
    bundle += Math.ceil(size/cloth);
}

let pen_bundle = Math.floor(n/pen);
let pen_else = n%pen;

console.log(bundle);
console.log(`${pen_bundle} ${pen_else}`);

File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.py
import math

n = int(input())
shirts = list(map(int, input().split()))
t, p = map(int, input().split())

ans1 = 0

for i in shirts:
    ans1 += math.ceil(i/t)

ans2 = n//p
ans3 = n%p

print(f"{ans1}")
print(f"{ans2} {ans3}")